### ISSUE TITLE
[issue]:  ventoy can't boot any iso on Dell Inspiron 3558, but can boot on Dell 3543

### ISSUE THREAD

**Issue by oldman20** [ventoy/Ventoy#1718]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.78

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

GPT

### Disk Capacity

256Gb

### Disk Manufacturer

Plextor

### Image file checksum (if applicable)

Already check

### What happened?

`

https://user-images.githubusercontent.com/39668756/178012169-092b5387-5e9a-4848-8298-b013fa1b1e21.MOV
Can boot into Ventoy, but can't boot any iso!
I tried also with  Sony pendriver 4Gb and GPT/MBR scheme, with/ without secure boot support but still not working. 
these are work flawlessly with Dell3543
Thank great work

**Comment by steve6375** [comment]
Did you try Ctrl+r first?

**Comment by oldman20** [comment]
> Did you try Ctrl+r first?

I tried but same result

**Comment by steve6375** [comment]
So does the 3558 boot to an OS OK. Is it working correctly? Passes a memory test? Boots from a Rufus-prepared USB drive?
Since you say the same Ventoy USB drive works on a different Dell, it sounds like a problem with the 3558?
Did you try a 'reset to defaults'. Try any other boot manager/loader like Easy2Boot? If both Legacy and UEFI booting to Ventoy show the same issue, maybe it has a RAM or memory map issue. Can you boot ANY small ISOs - even if you have to press F1 first to make them load into memory first?
See also https://github.com/ventoy/Ventoy/issues/1515 for memory map commands.

**Comment by oldman20** [comment]
> So does the 3558 boot to an OS OK. Is it working correctly? Passes a memory test? Boots from a Rufus-prepared USB drive? Since you say the same Ventoy USB drive works on a different Dell, it sounds like a problem with the 3558? Did you try a 'reset to defaults'. Try any other boot manager/loader like Easy2Boot? If both Legacy and UEFI booting to Ventoy show the same issue, maybe it has a RAM or memory map issue. Can you boot ANY small ISOs - even if you have to press F1 first to make them load into memory first? See also #1515 for memory map commands.

Yes, it boot win 11 and working flawlessly well. I can confirm RAM status is fine. Yes install Win 11 from rufus prepared USB drive!
I think maybe smt in BIOS Dell Inspiron 3558 i5200U
Before this post new thread github, i already tried reset bios to default, or update to latest bios version A21
I tried Clover bootloader too. Both Legacy and UEFI same issue
" Can you boot ANY small ISOs - even if you have to press F1 first to make them load into memory first? See also #1515 for memory map commands."
Didn't try it, i will trying one more time to see result. Thank!

**Comment by oldman20** [comment]
https://user-images.githubusercontent.com/39668756/178133121-b34d4645-c3c6-4bcc-a41d-4c93573ece2a.MOV
I tried ctrl+r but only isos below 20Mb said error no valid uefi (of course for hdd generation iso legacy) and with iso among  600Mb and up like video result

**Comment by steve6375** [comment]
So, just to be clear - you can successfully legacy boot to Ventoy and run any ISO on other systems, but if you try to legacy boot from the SAME VENTOY USB DRIVE on the 3558 then it fails? By legacy boot, I dont mean run Clover, I mean legacy boot from the 3558 BIOS so that Ventoy displays 'BIOS' in the menu at the bottom of the screen and then run an ISO.
Can you report the output when looking at the memory map pls?

**Comment by oldman20** [comment]
> So, just to be clear - you can successfully legacy boot to Ventoy and run any ISO on other systems, but if you try to legacy boot from the SAME VENTOY USB DRIVE on the 3558 then it fails? By legacy boot, I dont mean run Clover, I mean legacy boot from the 3558 BIOS so that Ventoy displays 'BIOS' in the menu at the bottom of the screen and then run an ISO. Can you report the output when looking at the memory map pls?

Yes, same VENTOY USBDRIVER work flawlessly on others system (Dell Inspiron 3543 both Legacy and UEFI)!
How can i report output when looking at the memory map? sr

**Comment by steve6375** [comment]
https://github.com/ventoy/Ventoy/issues/1515#issuecomment-1114664831
Also the VT command output, see comment a bit further down.
P.S. Your pictures show UEFI boot, not legacy boot?

**Comment by steve6375** [comment]
Its rather worrying that you cannot legacy boot to the Ventoy menu system (I presume you can legacy boot the same Ventoy USB drive on a different system?).

The map looks OK, I think (at least for first 4GB or so).
What about the VT command I asked about?
![image](https://user-images.githubusercontent.com/1469350/178157954-69b25048-3f44-4306-bf52-505a8fa92140.png)

**Comment by oldman20** [comment]
> Its rather worrying that you cannot legacy boot to the Ventoy menu system (I presume you can legacy boot the same Ventoy USB drive on a different system?).
> 
> The map looks OK, I think (at least for first 4GB or so). What about the VT command I asked about? ![image](https://user-images.githubusercontent.com/1469350/178157954-69b25048-3f44-4306-bf52-505a8fa92140.png)

i can confirm legacy boot the same Ventoy USB Drive well on different system (Dell Inspiron 3543)
sorry, what is VT Command u asked? vt_efi_dump_rsv_page?

**Comment by steve6375** [comment]
the vt_efi_dump_rsv_page command is typed in the grub2 console - just press c in Ventoy menu and type the command as in my screenshot. Same as you did for the lsmmap command!

Memtest86 is pretty small and should run OK on UEFI64. 

If it wasnt for the fact that it boots to Windows 10 Setup from a Rufus drive I would say the 3558 had a fault.
If it wasnt for the fact that other systems can run ISOs from the same Ventoy USB drive, I would say the Ventoy USB drive was corrupt.

What exactly is the USB drive you have **Ventoy** on? You say 256GB Plextor? - which I think is a SATA SSD drive - so what USB adaptor are you using? Did you try all USB ports (incl. USB 2 ports)?

**Comment by oldman20** [comment]
![20220711_021515](https://user-images.githubusercontent.com/39668756/178158835-aad8d276-8a3c-4209-9275-8ccbffc94d34.jpg)
Here is vt_efi_dump_rsv_page output
I switched to Sony Usb pen drive to test
And Plextor 256G covered in HDD Box 3.0, yes
I think smt fault with Dell3558

**Comment by steve6375** [comment]
smt? Surface Mount Technology component? But you said it boots to Win10 and you can install Win10 using Rufus, so why do you think it has a hardware fault?
Did you try USB 2 port for ventoy USB drive?

**Comment by oldman20** [comment]
https://user-images.githubusercontent.com/39668756/178159184-f44996f1-3874-44f0-b1f5-eba33cd1bcc7.mp4

I tried with usb2.0 still cant boot Legacy

**Comment by oldman20** [comment]
> smt? Surface Mount Technology component? But you said it boots to Win10 and you can install Win10 using Rufus, so why do you think it has a hardware fault? Did you try USB 2 port for ventoy USB drive?

Something, sorry:(

**Comment by steve6375** [comment]
I meant try both USB 2.0 ports and UEFI boot and load Memtest86 if possible.
If that still fails then sounds like a bad 3558 and I am afraid I don't know what the problem is...

**Comment by oldman20** [comment]
> I meant try both USB 2.0 ports and UEFI boot and load Memtest86 if possible. If that still fails then sounds like a bad 3558 and I am afraid I don't know what the problem is...
Dear  Dev, i tried usb2 port aand UEFI booting, with/ without memdisk, grub2 or combine they but still didn't work, duno why 

https://user-images.githubusercontent.com/39668756/178225540-4bcc1632-67f0-4d75-825b-0da563c3db2d.mp4


https://user-images.githubusercontent.com/39668756/178225545-a5cbbe0c-e248-45e4-be10-44a0a5778f4f.mp4


https://user-images.githubusercontent.com/39668756/178225558-4f93d248-9f5d-4cdd-8ad8-2df75260e3c7.mp4



https://user-images.githubusercontent.com/39668756/178225516-50f325fb-6e0f-46ec-abb8-bea07929389f.mp4

**Comment by ventoy** [comment]
@oldman20 
1. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in normal mode and report a video here.
2. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in grub2 mode (ctrl+r) and report a video here.
3. Have you ever booted into Ventoy menu in 3558 Legacy BIOS mode?
   (It will show  `BIOS    www.ventoy.net` in the bottom left not `UEFI    www.ventoy.net`)

**Comment by oldman20** [comment]
> @oldman20
> 
> 1. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in normal mode and report a video here.
> 2. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in grub2 mode (ctrl+r) and report a video here.
> 3. Have you ever booted into Ventoy menu in 3558 Legacy BIOS mode?
>    (It will show  `BIOS    www.ventoy.net` in the bottom left not `UEFI    www.ventoy.net`)

thank, i will. But i can use pop-os_22.04_amd64_nvidia_10.iso or linuxmint-20.3-cinnamon-64bit.iso instead ubuntu-22.04-desktop-amd64.iso ?
3. Nope, My dell3558 Bios V21 cant boot any usb (at least 3USB I have) from Legacy, strange!

**Comment by ventoy** [comment]
1. `linuxmint-20.3-cinnamon-64bit.iso ` is also OK.
2. Did you test both Ventoy in MBR partition style and Ventoy in GPT partition style? Normally, you should use MBR partition style for Legacy BIOS.

**Comment by oldman20** [comment]
> 1. `linuxmint-20.3-cinnamon-64bit.iso ` is also OK.
> 2. Did you test both Ventoy in MBR partition style and Ventoy in GPT partition style? Normally, you should use MBR partition style for Legacy BIOS.

i tested Venty in MBR style and only Memtest iso booteable
![0](https://user-images.githubusercontent.com/39668756/178648127-ed2623f6-ee9f-4ceb-bbef-9b890746f9c6.jpg)

https://user-images.githubusercontent.com/39668756/178648221-ae2b1978-f9c0-41ae-95ff-b49e6d69f5c4.mp4


https://user-images.githubusercontent.com/39668756/178648237-a1e3f02f-7624-45fc-8a09-2375f9fd0a4f.mp4

https://user-images.githubusercontent.com/39668756/178648714-b1031a42-9776-4e74-9a52-5b26d34

https://user-images.githubusercontent.com/39668756/178649011-8971ca0b-1e5e-421e-ab6f-8655a5dfd9ad.mp4

f2377.mp4

**Comment by oldman20** [comment]
> @oldman20
> 
> 1. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in normal mode and report a video here.
> 2. In 3558 UEFI mode boot a standard ubuntu iso file `ubuntu-22.04-desktop-amd64.iso` in grub2 mode (ctrl+r) and report a video here.
> 3. Have you ever booted into Ventoy menu in 3558 Legacy BIOS mode?
>    (It will show  `BIOS    www.ventoy.net` in the bottom left not `UEFI    www.ventoy.net`)
I tested with UEFI and iso LinuxMint created by rufus in normal mode, but with grub2 i dont know how to do, ctrl+r not working, maybe linux mint bootloader not exist that?
![11](https://user-images.githubusercontent.com/39668756/178649497-7b9cf758-32df-4ef3-9719-f370a265a8e4.jpg)
![1](https://user-images.githubusercontent.com/39668756/178649500-0875bece-94fc-4abc-9fa6-c980ce509b63.jpg)
Video is long and bigger than 10Mb so i cut waste timelap
![photo_2022-07-13_11-30-44](https://user-images.githubusercontent.com/39668756/178651067-49cb6a0b-2bc0-4cef-bc42-b1c6e4f53ee4.jpg)

https://user-images.githubusercontent.com/39668756/178650967-e19ecfe5-1608-48b5-8186-a3ea0ddf83f3.mp4

**Comment by oldman20** [comment]
1. Sr, i reformat Plextor Ssd and test with MBR layout, it working but not for all isos i have success on other machine before (dell inspiron 3543) 
2. and i realized when boot UEFI has this warning:
![Screenshot_20220715-163014_Gallery](https://user-images.githubusercontent.com/39668756/179196532-979622f3-c7a4-4e3e-ad0b-f48d30e32f93.jpg)

3. With all MBR and GPT layout, only some isos can boot (not like on dell3543) and here reasult like u required:
4.

**Comment by oldman20** [comment]
So it not reason for booting iso issue? Actually still dont know why it happened
